conn.go

```diff
@@ -151,31 +162,29 @@ func (c *conn) write(header parser.Header, args []reflect.Value) {
 }
 
 func (c *conn) onError(namespace string, err error) {
-	onErr := errorMessage{
-		namespace: namespace,
-		error:     err,
-	}
 	select {
-	case c.errorChan <- onErr:
+	case c.errorChan <- newErrorMessage(namespace, err):
 	case <-c.quitChan:
 		return
 	}
 }
 
-func (c *conn) parseArgs(types []reflect.Type) ([]reflect.Value, error) {
-	return c.decoder.DecodeArgs(types)
-}
-
 func (c *conn) serveError() {
 	defer c.Close()
+
 	for {
 		select {
 		case <-c.quitChan:
 			return
-		case msg := <-c.errorChan:
-			if handler := c.namespace(msg.namespace); handler != nil {
+		case err := <-c.errorChan:
+			errMsg, ok := err.(errorMessage)
+			// todo add log
```

Comment:
Fix todo?

Reply:
I added new (issue)[https://github.com/googollee/go-socket.io/issues/389] to implemented logger in debug mode. I will catch same errors in logger and print on stdout. By first step. But i doesn't add in this MR because it so big (
I sugget chose (logrus)[https://github.com/sirupsen/logrus]